File: problem-01/c/main.c
#include <stdio.h>

int solution_1(int n) {
	int output = 0;

	for (int i = 0; i < n; i++) {
		if (i % 3 == 0 || i % 5 == 0) {
			output += i;
		}
	}

	return output;
}

int main() {
	printf("Solution #1: %d\n", solution_1(1000));
}

File: problem-01/rust/src/main.rs
fn main() {
    println!("Solution #1: {}", solution_1(1000));
}

fn solution_1(n: u32) -> u32 {
    let mut output = 0;

    for x in 0..n {
        if x % 3 == 0 || x % 5 == 0 {
            output += x;
        }
    }

    return output
}
